airflow/models/dag.py

```diff
@@ -3902,6 +3901,16 @@ def get_dataset_triggered_next_run_info(self, *, session=NEW_SESSION) -> dict[st
             return None
         return get_dataset_triggered_next_run_info([self.dag_id], session=session)[self.dag_id]
 
+    @staticmethod
```

Comment:
Why does this need to be a static method? I think we can just have an instance method.

Reply:
removed it